include/nccl_ofi_tuner.h

```diff
@@ -89,16 +89,19 @@ struct nccl_ofi_tuner_model_dims {
 	int num_nodes;
 };
 
+typedef float nccl_ofi_tuner_cost_table_t[NCCL_NUM_FUNCTIONS][NCCL_NUM_ALGORITHMS][NCCL_NUM_PROTOCOLS];
+
 struct nccl_ofi_tuner_context {
     struct nccl_ofi_tuner_model_dims dims;
 	struct nccl_ofi_tuner_model_params model_params;
 
-	float base_costs[NCCL_NUM_FUNCTIONS][NCCL_NUM_ALGORITHMS][NCCL_NUM_PROTOCOLS];
+	nccl_ofi_tuner_cost_table_t base_costs;
 };
 
 /* Modeling functions */
 void nccl_ofi_tuner_model_costs(struct nccl_ofi_tuner_context *ctx);
 float nccl_ofi_tuner_compute_cost(struct nccl_ofi_tuner_model_params *params, struct nccl_ofi_tuner_model_dims *dims,
+								  const nccl_ofi_tuner_cost_table_t *base_costs,
```

Comment:
Nit: Indenting is off

Reply:
please review and approve https://github.com/aws/aws-ofi-nccl/pull/351 so we don't have to do this